panic: assert: missed arguments

Crash using commit `ab8c054a7179072bb12fa30c94dbb28f008c28d3` from 28/12.

```
INFO[12-30|01:04:15.344] [4/6 Execution] Done                     blk=21511835 blks=44 blk/s=0.2 txs=6927 tx/s=39 gas/s=3.70M buf=153.5MB/512.
0MB stepsInDB=0.00 step=1715.3 inMem=false alloc=11.8GB sys=22.7GB
panic: assert: missed arguments

goroutine 2556430 [running]:
github.com/erigontech/erigon-lib/state.(*AggregatorRoTx).TxNumsInFiles(...)
        github.com/erigontech/erigon-lib@v0.0.0-00010101000000-000000000000/state/aggregator.go:935
github.com/erigontech/erigon-lib/state.(*AggregatorRoTx).StepsInFiles(...)
        github.com/erigontech/erigon-lib@v0.0.0-00010101000000-000000000000/state/aggregator.go:926
github.com/erigontech/erigon-lib/state.(*AggregatorRoTx).PruneSmallBatches(0xc27f470cc0, {0x345c338, 0xc001b90b90}, 0x20bde7364000, {0x3490520
, 0xc2884135c0})
        github.com/erigontech/erigon-lib@v0.0.0-00010101000000-000000000000/state/aggregator.go:1153 +0xc30
github.com/erigontech/erigon/eth/stagedsync.ExecV3({_, _}, _, {_, _}, _, {{0x3474410, 0xc000d56690}, 0x20000000, {0x1, ...}, ...}, ...)
        github.com/erigontech/erigon/eth/stagedsync/exec3.go:692 +0x3a45
github.com/erigontech/erigon/eth/stagedsync.ExecBlockV3(_, {_, _}, {{_, _}, {_, _}, _}, _, {0x345c338, ...}, ...)
        github.com/erigontech/erigon/eth/stagedsync/stage_execute.go:162 +0x20f
github.com/erigontech/erigon/eth/stagedsync.SpawnExecuteBlocksStage(_, {_, _}, {{_, _}, {_, _}, _}, _, {0x345c338, ...}, ...)
        github.com/erigontech/erigon/eth/stagedsync/stage_execute.go:256 +0x10e
github.com/erigontech/erigon/eth/stagedsync.PipelineStages.func10(0x9?, 0x7b7881f55190?, {0x3455d60?, 0xc1984c2b00?}, {{0x3490520, 0xc2884135c
0}, {0x0, 0x0}, 0x0}, {0x3472da8, ...})
        github.com/erigontech/erigon/eth/stagedsync/default_stages.go:238 +0xf0
github.com/erigontech/erigon/eth/stagedsync.(*Sync).runStage(0xc1984c2b00, 0xc1984ca640, {0x3474410, 0xc000d56690}, {{0x3490520, 0xc2884135c0}
, {0x0, 0x0}, 0x0}, 0x0, ...)
        github.com/erigontech/erigon/eth/stagedsync/sync.go:528 +0x190
github.com/erigontech/erigon/eth/stagedsync.(*Sync).Run(0xc1984c2b00, {0x3474410, 0xc000d56690}, {{0x3490520, 0xc2884135c0}, {0x0, 0x0}, 0x0},
 0x71?, 0x0)
        github.com/erigontech/erigon/eth/stagedsync/sync.go:412 +0x2ad
github.com/erigontech/erigon/turbo/execution/eth1.(*EthereumExecutionModule).updateForkChoice(0xc190ec3400, {0x345c338, 0xc001b90b90}, {0x46,
0x94, 0x67, 0xd1, 0xc7, 0x18, 0x30, ...}, ...)
        github.com/erigontech/erigon/turbo/execution/eth1/forkchoice.go:447 +0x1465
created by github.com/erigontech/erigon/turbo/execution/eth1.(*EthereumExecutionModule).UpdateForkChoice in goroutine 2557083
        github.com/erigontech/erigon/turbo/execution/eth1/forkchoice.go:130 +0x31c
```

--- a/erigon-lib/state/aggregator.go
+++ b/erigon-lib/state/aggregator.go
@@ -1056,7 +1056,7 @@ func (ac *AggregatorRoTx) PruneSmallBatchesDb(ctx context.Context, timeout time.
 				ac.a.logger.Info("[snapshots] pruning state",
 					"until commit", time.Until(started.Add(timeout)).String(),
 					"pruneLimit", pruneLimit,
-					"aggregatedStep", ac.StepsInFiles(),
+					"aggregatedStep", ac.StepsInFiles(kv.StateDomains...),
 					"stepsRangeInDB", ac.a.StepsRangeInDBAsStr(tx),
 					"pruned", fullStat.String(),
 				)
@@ -1150,7 +1150,7 @@ func (ac *AggregatorRoTx) PruneSmallBatches(ctx context.Context, timeout time.Du
 			ac.a.logger.Info("[snapshots] pruning state",
 				"until commit", time.Until(started.Add(timeout)).String(),
 				"pruneLimit", pruneLimit,
-				"aggregatedStep", ac.StepsInFiles(),
+				"aggregatedStep", ac.StepsInFiles(kv.StateDomains...),
 				"stepsRangeInDB", ac.a.StepsRangeInDBAsStr(tx),
 				"pruned", fullStat.String(),
 			)

